src/utils/Logger.ts

```diff
@@ -146,30 +151,99 @@ class Logger {
     return error
   }
 
-  getReactNativeLogsFilePath = () => {
-    return RNFS.CachesDirectoryPath + '/rn_logs.txt'
+  getReactNativeLogFilePath = () => {
+    return `${this.getReactNativeLogsDir()}/${format(new Date(), 'yyyy-MMM')}.txt`
   }
 
-  getLogs = async () => {
-    // TODO(Rossy) Does this technique of passing logs back as a string
-    // fail when the logs get too big?
+  getReactNativeLogsDir = () => {
+    return `${RNFS.CachesDirectoryPath}/rn_logs`
+  }
+
+  getLogsToAttach = async () => {
+    try {
+      const logDir = this.getReactNativeLogsDir()
+      const logFiles = await RNFS.readDir(logDir)
+      const path = Platform.OS === 'ios' ? RNFS.TemporaryDirectoryPath : RNFS.ExternalDirectoryPath
+      const toAttach = []
+      for (const file of logFiles) {
+        // Always remove the logs from the current month and re-copy
+        // But not the logs from previous months if they exist
+        if (file.name === `${format(new Date(), 'yyyy-MMM')}.txt`) {
+          await RNFS.unlink(`${path}/${file.name}`)
+          await RNFS.copyFile(file.path, `${path}/${file.name}`)
+        } else if (!(await RNFS.exists(`${path}/${file.name}`))) {
+          await RNFS.copyFile(file.path, `${path}/${file.name}`)
```

Comment:
Why do we need to do this file copy stuff?  Why not attempt to attach the files directly?

Reply:
It's to do file permissions. We need to move them to a directory the mail client would have access to. I know this is the case for Android, but maybe we can omit for iOS. @jeanregisser

Reply:
Testing with iOS the above was also needed to have access to the files.